.travis.yml

```diff
@@ -13,6 +13,7 @@ matrix:
   fast_finish: true
   allow_failures:
     - rust: nightly
+    - rust: beta
```

Comment:
So now we know beta works on Xenia’s, let’s make it required again to prevent problems with automatic upgrades of stable.